src/syft/core/common/message.py

```diff
@@ -65,76 +73,65 @@ def __init__(
         self.cached_deseralized_message = None
 
     @property
-    def message(self):
+    def message(self) -> "SyftMessage":
         if self.cached_deseralized_message is None:
             self.cached_deseralized_message = _deserialize(
                 blob=self.serialized_message, from_binary=True
             )
         return self.cached_deseralized_message
 
-
     @property
     def is_valid(self) -> bool:
-
         try:
-            verify_key: VerifyKey = self.verify_key
-            _ = verify_key.verify(self.serialized_message, self.signature)
+            _ = self.verify_key.verify(self.serialized_message, self.signature)
         except BadSignatureError:
             return False
 
         return True
 
-    #
```

Comment:
good call on removing this finally - i wasn't sure which is why i left it in comments